src/components/YourCompany/OfficeAddresses.vue

```diff
@@ -133,63 +131,65 @@
     <!-- Addresses Edit -->
     <v-card flat v-else>
       <ul class="list address-list">
-        <div class="address-edit-header">
-          <label class="address-edit-title">Registered Office</label>
-        </div>
+        <div id="edit-registered-address">
+          <div class="address-edit-header">
+            <label class="address-edit-title">Registered Office</label>
+          </div>
 
-        <!-- Registered Mailing Address -->
-        <li class="ma-5">
-          <div class="meta-container">
-            <label>Mailing Address</label>
-            <div class="meta-container__inner">
-              <div class="address-wrapper">
-                <base-address ref="regMailingAddress"
-                  id="address-registered-mailing"
-                  :address="mailingAddress"
-                  :editing="true"
-                  :schema="addressSchema"
-                  @update:address="updateAddress('mailingAddress', mailingAddress, $event)"
-                  @valid="updateValidity('mailingAddress', $event)"
-                />
+          <!-- Registered Mailing Address -->
+          <li class="ma-5">
+            <div class="meta-container">
+              <label>Mailing Address</label>
+              <div class="meta-container__inner">
+                <div class="address-wrapper">
+                  <base-address ref="regMailingAddress"
+                    id="address-registered-mailing"
+                    :address="mailingAddress"
+                    :editing="true"
```

Comment:
Is this meant to be always `TRUE`? Sort of a mode selector like we had with `isSummary` in the create UI?

Reply:
Yes. Here, we are using this component in a particular layout for editing an address. In another place(s) we use this component in a different layout for displaying (read only) an address.